modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyQuery.jsx

```diff
@@ -95,7 +95,7 @@ function VocabularyQuery(props) {
           event.preventDefault();
         } else if (event.key == 'ArrowDown') {
           // Move the focus to the suggestions list
-          if (menuRef.children.length > 0) {
+          if (menuRef?.children?.length > 0) {
             menuRef.children[0].focus();
```

Comment:
```suggestion
          if (menuRef?.current?.children?.length > 0) {
            menuRef.current.children[0].focus();
```

Huh, why do we always focus the first one? Will moving the focus make future arrow keys navigate the list?

Reply:
Even with this change, arrow navigation still doesn't work.

Reply:
Works for me with this fix